atlas

```diff
@@ -473,6 +473,12 @@ git_sparse_checkout_set() {
     xargs git sparse-checkout set
 }
 
+git_disk_usage() {
+  # Print the disk usage of the whole git "atlas pull" directory
+  # Minor note: the size are essentially rounded up to the next 512-byte value. See: https://superuser.com/a/402000
+  du -s . | grep -o '[0-9]*'
```

Comment:
[suggestion] You could use `du -s -k .` to print the size in kilobytes if 512-byte blocks are too awkward to read. (Could also add `--apparent-size` although I could go either way on that one.)